content/md/en/docs/tutorials/work-with-pallets/use-macros-in-a-custom-pallet.md

```diff
@@ -119,41 +119,33 @@ Therefore, the first step is to remove some files and content from the files in
 1. Add the following code that includes a minimal skeleton for a FRAME pallet and its required [macros](https://paritytech.github.io/substrate/master/frame_support/attr.pallet.html):
 
    ```rust
-   // Re-export pallet items so that they can be accessed from the crate namespace.
-   pub use pallet::*;
-
-   #[frame_support::pallet]
-   pub mod pallet {
-   	use frame_support::{pallet_prelude::*, storage::bounded_vec::BoundedVec};
-   	use frame_system::pallet_prelude::*;
-
-   	// Define the pallet struct placeholder, various pallet function are implemented on it.
-   	#[pallet::pallet]
-   	#[pallet::generate_store(pub(super) trait Store)]
-   	pub struct Pallet<T>(_);
-
-   	// TODO: Update the `Config` block below
-   	/// Configure the pallet by specifying the parameters and types on which it depends.
-   	#[pallet::config]
-   	pub trait Config: frame_system::Config {
-   		/// Because this pallet emits events, it depends on the runtime's definition of an event.
-   		type Event: From<Event<Self>> + IsType<<Self as frame_system::Config>::Event>;
-   	}
-
-   	// TODO: Update the `event` block below
-   	#[pallet::event]
-   	#[pallet::generate_deposit(pub(super) fn deposit_event)]
-   	pub enum Event<T: Config> {
-   		/// Event documentation should end with an array that provides descriptive names for event
-   		/// parameters. [something, who]
-   		SomethingHappened(),
-   	}
-   	// TODO: add #[pallet::error] block
-
-   	// TODO: add #[pallet::storage] block
-
-   	// TODO: add #[pallet::call] block
-   }
```

Comment:
@sacha-l this was intentional, not formatting :grimacing:

Reply:
specifically, this code ***compiles*** - so we need `Event` defined, though we replace it in a moment